src/legacy/components/site-header/examples/logged-in-with-content.html

```diff
@@ -39,7 +39,8 @@
           class="site-branding--logo"
           src="https://mltshp-cdn.com/static/images/logo-compact.svg"
           alt="MLTSHP"
-        /><span class="site-branding--icon" />
+        />
+        <div class="site-branding--icon" />
```

Comment:
Oh, actually, should this have a label or anything for accessibility?

Reply:
I mean on the actual site, of course. We do have an a tag around it, but it doesn't have any labeling either.

Reply:
Can't hurt!